src/main/io/serial_msp.c

```diff
@@ -1721,6 +1725,47 @@ static bool processInCommand(void)
         isRebootScheduled = true;
         break;
 
+#ifdef USE_SERIAL_1WIRE
+    case MSP_SET_1WIRE:
+        // get channel number
+        i = read8();
+        // we do not give any data back, assume channel number is transmitted OK
+        if (i==0xFF) {
+            // 0xFF -> preinitialize the Passthrough
+            // switch all motor lines HI
+            usb1WireInitialize();
+            // and come back rigth afterwards
+            // rem: App: Wait at least appx. 500 ms for BLHeli to jump into bootloader mode before try to connect any ESC
```

Comment:
Preferable to wrap comment here onto a new line after c.80 characters, so easier to read.